src/cljs/org/broadinstitute/firecloud_ui/page/library/library_page.cljs

```diff
@@ -0,0 +1,120 @@
+(ns org.broadinstitute.firecloud-ui.page.library.library_page
+  (:require
+    [dmohs.react :as react]
+    [org.broadinstitute.firecloud-ui.common :as common]
+    [org.broadinstitute.firecloud-ui.common.components :as comps]
+    [org.broadinstitute.firecloud-ui.common.icons :as icons]
+    [org.broadinstitute.firecloud-ui.common.modal :as modal]
+    [org.broadinstitute.firecloud-ui.endpoints :as endpoints]
+    [org.broadinstitute.firecloud-ui.common.style :as style]
+    [org.broadinstitute.firecloud-ui.common.table :as table]
+    [org.broadinstitute.firecloud-ui.common.table-utils :refer [float-right]]
+    [org.broadinstitute.firecloud-ui.nav :as nav]
+    [org.broadinstitute.firecloud-ui.utils :as utils]
+    ))
+
+
+(react/defc DatasetsTable
+  {
+   :render
+   (fn [{:keys [state this]}]
+     (cond
+       (:error-message @state) (style/create-server-error-message (:error-message @state))
+       :else
+       [:div {:style {:flex "2 0 auto" :marginRight "1em"}}
+        [:div {:style {:fontWeight 700 :fontSize "125%"}} "Search Results: "
+         [:span {:style {:fontWeight 100}} (str (:total @state) " Datasets found")]]
+        [:div {:style {:padding "4"}}]
+        [table/Table
+         {:ref "table"
+          :header-row-style {:fontWeight 500 :fontSize "90%"
+                             :backgroundColor nil
+                             :color "black"
+                             :borderBottom (str "2px solid " (:border-light style/colors))
+                             }
+          :header-style {:padding "0.5em 0"}
+          :resizable-columns? false :reorderable-columns? true ;:resize-tab-color (:line-default style/colors)
+          :body-style {:fontSize nil :fontWeight nil}
+          :row-style {:backgroundColor nil :height 22} ;:height row-height-px :borderTop style/standard-line}
+          :cell-content-style {:padding nil
+                               :color (:text-light style/colors)
+                               :fontSize "90%"
+                               }
+          :filterable? false
+          :columns [{:header "Dataset Name" :starting-width 300
+                     :sort-by clojure.string/lower-case
```

Comment:
I don't think this is right.  Since you're rendering `library:datasetName` I think you need to sort on that too.